integrations/close.io/setup.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+var _ = require('lodash');
+
+exports.createCollections = function (Implementation, apimap, collectionName) {
+  var collectionDisplayName = _.capitalize(collectionName);
+
+  // jshint camelcase: false
+  apimap.push({
+    name: collectionName + '_closeio_leads',
+    displayName: collectionDisplayName + ' Leads',
+    icon: 'closeio',
+    isVirtual: true,
+    isReadOnly: true,
+    fields: [
+      { field: 'url', type: 'String', isSearchable: false, widget: 'link' },
+      { field: 'display_name', type: 'String', isSearchable: false },
+      { field: 'status_label', type: 'String' },
+      { field: 'created_by_name', type: 'String', isSearchable: false },
+      { field: 'date_updated', type: 'Date', isSearchable: false },
+      { field: 'date_created', type: 'Date', isSearchable: false },
+      { field: 'description', type: 'String', isSearchable: false },
+      { field: 'emails', type: ['String'], reference: collectionName +
+        '_closeio_emails.id' },
+    ]
+  });
+
+  apimap.push({
+    name: collectionName + '_closeio_emails',
+    displayName: collectionDisplayName + ' Conversations',
+    icon: 'closeio',
+    isVirtual: true,
+    isReadOnly: true,
+    onlyForRelationships: true,
+    fields: [
+      { field: 'status', type: 'String', isSearchable: false },
+      { field: 'sender', type: 'String', isSearchable: false },
+      { field: 'subject', type: 'String', isSearchable: false },
+      { field: 'body_text', type: 'String', isSearchable: false }
+    ]
+  });
+};
+
+exports.createFields = function (implementation, model, schema,
+  collectionName) {
+
+  schema.fields.push({
+    field: 'closeio_lead',
+    type: 'String',
+    reference: collectionName + '_closeio_leads.id',
+    column: null,
+    isSearchable: false,
+    integration: 'close.io'
+  });
+
```

Comment:
• Agreed with `createField` name.
• Yes, it's not the same as `refund` action on Stripe. The `Create Close.io lead` is added on an existing collection (`mapping`). The `refund` action is added on a smart collection (`stripe.payments`)